src/libs/deploy.js

```diff
@@ -178,11 +180,131 @@ export const deployAppleStore = async (signedApp, workspace = '/tmp/') => {
 /**
  * AirWatch Deployment
  *
- * @param {String} signedApp The name of the signed app
+ * @param {String} job The name of the signed app
+ * @param {String} platform Define if it's android or iOS app
+ * @param {String} awOrgID The airwatch organization group ID to distribute the app
+ * @param {String} awFileName The name of the app to appear in airwatch
  * @param {string} [workspace='/tmp/'] The workspace to use
  * @returns The status of the deployment
  */
-// eslint-disable-next-line no-unused-vars
-export const deployAirWatch = async (signedApp, workspace = '/tmp/') => {
-  // TODO
+// eslint-disable-next-line import/prefer-default-export
+export const deployAirWatch = async (signedApp, platform, awOrgID, awFileName, workspace = '/tmp/') => {
+  /*
+  Values for airwatch api v8_1:
+    DeviceType: android -> '5'; Apple -> '2'
+    ModelId: android -> 5; iPhone -> 1; iPad -> 2
+    ApplicationName: android -> apk; Apple -> ipa
+  */
+
+  let deviceType = '';
+  let applicationName = '';
+  let modelId = 0;
+
+  switch (platform) {
+    case 'ios':
+    {
+      applicationName = awFileName + '.ipa';
+      deviceType = '2';
```

Comment:
I think you have some Magic Numbers here. Please move them to a const file and use a descriptive var name so we have some idea what they are / do.

Reply:
It might require making requests to AirWatch api in the future to get the value, so hardcoding them here for now.

Reply:
:thumbsup: